<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLG_ContactPair</class>
 <widget class="QDialog" name="DLG_ContactPair">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Make contact pair</string>
  </property>
  <widget class="QTabWidget" name="TB_METHOD">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>328</width>
     <height>261</height>
    </rect>
   </property>
   <widget class="QWidget" name="T_DHS">
    <attribute name="title">
     <string>DHS</string>
    </attribute>
    <widget class="QGroupBox" name="GB_ContactPair">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>301</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Contact pair</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>281</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="L_FirstObject">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>First object</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="CB_FirstObject"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="L_SecondObject">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Second object</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="CB_SecondObject"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="GB_Parameter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>301</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Parameter</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>21</y>
        <width>281</width>
        <height>100</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="L_Restitution">
         <property name="text">
          <string>Restitution</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="LE_Restitution"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="L_StiffnessRatio">
         <property name="text">
          <string>Stiffness ratio</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="LE_StiffnessRatio"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="L_Friction">
         <property name="text">
          <string>Friction</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="LE_Friction"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="L_Cohesion">
         <property name="text">
          <string>Cohesion</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="LE_Cohesion">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="PB_Ok">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_Cancle">
      <property name="text">
       <string>Cancle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="L_Name">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_Name"/>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="CHB_IgnoreCondition">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>324</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ignore condition</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>320</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="L_IgnoreTime">
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_IgnoreTime"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>LE_Name</tabstop>
  <tabstop>TB_METHOD</tabstop>
  <tabstop>CB_FirstObject</tabstop>
  <tabstop>CB_SecondObject</tabstop>
  <tabstop>LE_Restitution</tabstop>
  <tabstop>LE_StiffnessRatio</tabstop>
  <tabstop>LE_Friction</tabstop>
  <tabstop>PB_Ok</tabstop>
  <tabstop>PB_Cancle</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
